### ISSUE TITLE
Needs to clarify IO traffic and MPI traffic effects in the text

### ISSUE THREAD

**Issue by mkhoshle** [hpcanalytics/paper-hpc-py-parallel-mdanalysis#13]
Need to clarify where IO traffic is affected by MPI traffic and where MPI traffic is affected by IO traffic in the related sections in the text.

**Comment by orbeckst** [comment]
Do you have a specific section (line numbers) in mind? My current read of the draft is that it contains a sufficient level of detail. If there's something where we can more clearly and succinctly comment on the data then that's good. But I would restrict myself to working with the data we have as opposed to offering more hypotheses – we already have a quite a few paragraphs where we propose a sensible explanation (that is corroborated by the literature) but where we don't necessarily have absolute direct evidence.

In short: I didn't see anything major that needs fixing in this regard but I'll wait for the specifics – maybe I overlooked something. I would, however, first focus on #15 and #11.

**Comment by mkhoshle** [comment]
Yes, I agree. Yesterday I was thinking that there might be some gaps. But when I went through the paper I did not find anything. So I close this issue.
